samples/sensor/ti_hdc/src/main.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019 Centaur Analytics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <sensor.h>
+#include <misc/printk.h>
+#include <misc/__assert.h>
+#include <logging/log.h>
+#include <gpio.h>
+
+void main(void)
+{
+	printk("Running on %s!\n", CONFIG_ARCH);
+	struct device *dev = device_get_binding(DT_TI_HDC_0_LABEL);
+
+	__ASSERT(dev != NULL, "Failed to get device binding");
+
+	printk("Dev %p name %s is ready!\n", dev, dev->config->name);
+
+	struct sensor_value temp, humidity;
+
+	while (1) {
+		/* take a sample */
+		printk("Fetching...\n");
+		sensor_sample_fetch(dev);
+		sensor_channel_get(dev, SENSOR_CHAN_AMBIENT_TEMP, &temp);
+		sensor_channel_get(dev, SENSOR_CHAN_HUMIDITY, &humidity);
+
+		/* print the result */
+		printk("Temp = %d.%06d C, RH = %d.%06d %%\n",
+		       temp.val1, temp.val2, humidity.val1, humidity.val2);
+		k_sleep(K_SECONDS(1));
```

Comment:
redundant?

Reply:
This sleep is just for not spamming stdout

Reply:
But below is the sleep for 10 seconds.

Reply:
Ah sorry didn't notice that it was a duplicate. Will be deleted

Reply:
Not addressed